src/libstd/env.rs

```diff
@@ -13,6 +13,13 @@
 //! This module contains functions to inspect various aspects such as
 //! environment variables, process arguments, the current directory, and various
 //! other important directories.
+//!
+//! There are several functions and structs in this module that have a
```

Comment:
Looks good to me; did you have something specific you were worried about, or just generally?